socorro/external/es/query.py

```diff
@@ -17,8 +19,16 @@
 from socorro.lib import datetimeutil, external_common
 
 
-class Query(object):
-    """Implement the /query service with ElasticSearch"""
+class Query(RequiredConfig):
```

Comment:
All middleware models get `config` passed to them. Therefore they should all derive from `RequiredConfig` and specify what configuration they're looking for. This fixes `Query` accordingly.